src/BlockEntities/BrewingstandEntity.cpp

```diff
@@ -111,7 +112,27 @@ bool cBrewingstandEntity::Tick(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 		Ingredient.m_ItemCount -= 1;
 		m_Contents.SetSlot(bsIngredient, Ingredient);
 
+		// Fuel slot
+		m_Fuel--;
+		if (m_Fuel == 0)
+		{
+			if (!m_Contents.GetSlot(bsFuel).IsEmpty())
+			{
+				cItem Fuel = m_Contents.GetSlot(bsFuel);
+				Fuel.m_ItemCount -= 1;
+				m_Contents.SetSlot(bsFuel, Fuel);
+				m_Fuel = 20;
```

Comment:
There's only one type of fuel: blaze powder (which always gives the same amount).  Nothing else can be put into the slot.